appveyor.yml

```diff
@@ -1,77 +1,5 @@
-platform: x64
```

Comment:
It'd be easier to delete it.

Reply:
Appveyor will error if it cannot find appveyor.yml. If you prefer deleting the code, you should turn Appveyor off in the settings.
https://github.com/atilaneves/dpp/settings